src/freertos_drivers/esp32/Esp32HardwareCanAdapter.hxx

```diff
@@ -236,7 +240,7 @@ private:
             }
 
             /// ESP32 native CAN driver frame
-            can_message_t msg = {0};
+            can_message_t msg = {};
```

Comment:
what is the reason for this change? Will the new code also zero initialize?

Reply:
Gcc complained on incomplete initializer. This is supposed to zero it as well but I have no problems putting in a memset instead of you prefer

Reply:
switching to memset instead.